gtools.json

```diff
@@ -0,0 +1,32 @@
+{
+    "homepage": "http://p-nand-q.com/download/gtools",
+    "version": "4.2",
+    "description": "Gtools include: pserv, dllusage, gkalk, as well as a couple of command-line only tools: regdiff, supershell, ddupe, pathed, touch, which, md5sums.",
+    "license": "New BSD License",
+    "hash": "sha1:7152656b8698daf80027861a2bedba236a3185df",
+    "url": "http://p-nand-q.com/download/gtools/gtools-4.2.exe#dl.7z",
+    "bin": [
```

Comment:
Add `"post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\" -Recurse",` to delete junk folder with installer files